src/BlurView.web.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component, PropTypes } from 'react';
+import { View } from 'react-native';
+import { overlayColorForProps, blurRadiusForProps } from './lib/ios_blur_view_emulation';
+
+// Note: Web works in a similar way to Android,
+// where you must pass a view to be blurred.
+// Unlike Android, the original view is blurred.
+// We accomplish this by updating the viewRef's style
+// when the BlurView is mounted, and removing the style
+// when it is unmounted.
+
+class BlurView extends Component {
+  componentWillMount() {
+    this.updateViewRefStyleProperties();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (
+      blurRadiusForProps(this.props) !== blurRadiusForProps(nextProps) ||
+      this.props.transitionDuration !== nextProps.transitionDuration
+    ) {
+      this.updateViewRefStyleProperties();
+    }
+  }
+
+  componentWillUnmount() {
+    this.removeViewRefStyleProperties();
+  }
+
+  updateViewRefStyleProperties() {
+    const { viewRef, transitionDuration } = this.props;
+    if (transitionDuration > 0) {
+      viewRef.style.setProperty(
+        'transition',
+        `filter ${transitionDuration}ms linear`,
+      );
+    }
+    const blurRadius = blurRadiusForProps(this.props);
+    viewRef.style.setProperty('filter', `blur(${blurRadius}px)`);
+  }
+
+  removeViewRefStyleProperties() {
+    this.props.viewRef.style.removeProperty('transition');
+    this.props.viewRef.style.removeProperty('filter');
+  }
+
+  render() {
+    const style = {
+      backgroundColor: overlayColorForProps(this.props),
+    };
+
+    return (
+      <View style={[style, this.props.style]}>
+        {this.props.children}
+      </View>
+    );
+  }
+}
+
+BlurView.propTypes = {
+  ...View.propTypes,
```

Comment:
Use `ViewPropTypes` which is exported from the packages: `import { ViewPropTypes } from 'react-native'`